common/proto/HStream/Server/HStreamApi.proto

```diff
@@ -337,7 +337,7 @@ message GetConnectorRequest {
 
 message Connector {
   string id = 1;
-  int32 status = 2;
+  string status = 2;
```

Comment:
this should be `enum` ?

Reply:
So we should use an `enum Status` type from grpc instead of the status type defined [here](https://github.com/hstreamdb/hstream/blob/0e91cb02b8b940ebc78bcde236697883eb4d9be8/hstream/src/HStream/Server/Persistence.hs#L76)? Or we keep both and implement converters from one to the other.